src/main/java/com/efficio/fieldbook/service/FieldbookServiceImpl.java

```diff
@@ -1042,36 +1042,6 @@ public void addConditionsToTrialObservationsIfNecessary(final Workbook workbook)
 		return possibleValues;
 	}
 
-	@Override
-	public void manageCheckVariables(final UserSelection userSelection, final ImportGermplasmListForm form) {
-		if (userSelection.getImportedCheckGermplasmMainInfo() != null && form.getImportedCheckGermplasm() != null) {
-			if (!form.getImportedCheckGermplasm().isEmpty()
-				&& !this.hasCheckVariables(userSelection.getWorkbook().getConditions())) {
-				// add check variables
-				this.addCheckVariables(userSelection.getWorkbook().getConditions(), form);
```

Comment:
I think you can also delete the ff methods:
- addCheckVariables
- updateCheckVariables
- deleteCheckVariables
- setCheckVariableToDelete
- hasCheckVariables
- isCheckVariable